server/blob/blob-api/src/main/java/org/apache/james/blob/api/MetricableBlobStore.java

```diff
@@ -33,6 +33,7 @@
 
     static final String BLOB_STORE_METRIC_PREFIX = "blobStore:";
     static final String SAVE_BYTES_TIMER_NAME = BLOB_STORE_METRIC_PREFIX + "saveBytes";
+    static final String SAVE_STRING_TIMER_NAME = BLOB_STORE_METRIC_PREFIX + "saveString";
```

Comment:
If you add one more metric, I'm worry you have to modify the grafana board to add this metric.